mysql-test/suite/rocksdb/t/update_with_keys.test

```diff
@@ -27,7 +23,7 @@ DROP TABLE t1;
 --echo # RocksDB: skip the test for secondary UNIQUE keys.
 --disable_parsing
 --error ER_GET_ERRMSG
-CREATE TABLE t1 (a INT, b CHAR(8), pk INT AUTO_INCREMENT PRIMARY KEY, UNIQUE INDEX(a)) ENGINE=innodb;
+CREATE TABLE t1 (a INT, b CHAR(8), pk INT AUTO_INCREMENT PRIMARY KEY, UNIQUE INDEX(a)) ENGINE=INNODB;
```

Comment:
How in the world did I miss that one...There are places MyISAM is explicitly used, and places where InnoDB was explicitly used where it should have been MyRocks that I fixed. This now has me questioning some of the work. I need to re-review this all again myself now that I have 'looked away' for a bit.